app/screens/home/search/results/header.tsx

```diff
@@ -44,15 +44,26 @@ const getStyleFromTheme = makeStyleSheetFromTheme((theme: Theme) => {
             borderColor: theme.centerChannelBg,
             marginTop: 2,
         },
-        buttonsContainer: {
+        header: {
             marginBottom: 12,
             paddingHorizontal: 12,
             flexDirection: 'row',
+            justifyContent: 'space-between',
```

Comment:
please show how is this modifying the UI and the impact, I know we spend a long time making this right, so I want to avoid changes here if they don't match the UI unless the  UI changes for this are expected.

Reply:
I made a screenshot on iOS and Android, and here's the link to Figma: https://www.figma.com/design/afCozEMCPeyfomstQfyp61/Search-Across-Teams?node-id=4274-35733&p=f&t=tlOZueT4DGgmwuhq-0

iPhone:
![Simulator Screenshot - iPhone 16 Pro - 2024-12-19 at 10 50 28](https://github.com/user-attachments/assets/7c033a27-31e8-4af9-bb81-bfcfe4785009)
Android:

![Screenshot_1734630717](https://github.com/user-attachments/assets/02d7ed87-efbe-490f-8595-7add29b68ae2)